packages/visx-demo/src/sandboxes/visx-xychart/Example.tsx

```diff
@@ -120,15 +137,25 @@ export default function Example({ height }: Props) {
                 data={data}
                 xAccessor={accessors.x.Austin}
                 yAccessor={accessors.y.Austin}
-                fillOpacity={0.5}
+                fillOpacity={0.4}
                 curve={curve}
               />
+              {!renderBarSeries && (
```

Comment:
this just makes it so we render 3 lines when we don't render a bar series, ditto on area below